<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PluginConfigDialog</class>
 <widget class="QDialog" name="PluginConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>104</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
  font: 12px &quot;Noto Sans&quot;, &quot;FreeSans&quot;;
  background: #3B3B3B;
  color: whitesmoke;
}

/* QLineEdit */

QLineEdit {
  background: #3B3B3B;
  border: 1px solid #CACACA;
  padding: 2px;
}
QLineEdit:focus {
  border: 1px solid #FFFFFF;
}

/* QPushButton */

QPushButton {
  background: #848484;
  border: none;
  border-radius: 4px;
  color: whitesmoke;
  min-width: 76px;
  min-height: 15px;
  padding: 3px 6px;
}
QPushButton:focus {
  border: 1px solid #D6D6D6;
  outline: none;
}
QPushButton:pressed {
  background: #676767;
}
QPushButton:disabled {
  background: transparent;
  border: 1px solid #6E6E6E;
  color: #6F6F6F;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Edit config items below, leave empty means unchanged</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="gridWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout"/>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
